packages/pilot/src/containers/RecipientTable/index.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import AddIcon from 'emblematic-icons/svg/Add32.svg'
+import IconInfo from 'emblematic-icons/svg/Info32.svg'
+import Search32 from 'emblematic-icons/svg/Search32.svg'
+
+import { pick } from 'ramda'
+
+import {
+  Alert,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardTitle,
+  Col,
+  Grid,
+  Input,
+  Pagination,
+  Row,
+  Table,
+} from 'former-kit'
+
+import Loader from '../../components/Loader'
+import style from './style.css'
+import Filter from '../Filter'
+import tableColumns from './tableColumns'
+
+const RecipientTable = ({
+  clearFilterDisabled,
+  confirmationDisabled,
+  expandedRows,
+  filterOptions,
+  loading,
+  onDetailsClick,
+  onExpandRow,
+  onFilterChange,
+  onFilterClear,
+  onFilterConfirm,
+  onOrderChange,
+  onPageChange,
+  onRowClick,
+  onSelectRow,
+  pagination: {
+    offset,
+    total,
+  },
+  push,
+  query,
+  rows,
+  selectedRows,
+  t,
+}) => {
+  const columns = tableColumns({
+    onDetailsClick,
+    t,
+  })
+  const handleOrderChange = columnIndex => onOrderChange(columns[columnIndex].accessor) // eslint-disable-line
```

Comment:
Poderia quebrar a linha e adicionar parênteses para remover esse `eslint-disable-line`:

```suggestion
  const handleOrderChange = columnIndex => (
    onOrderChange(columns[columnIndex].accessor)
  )
```